includes/class-amp-theme-support.php

```diff
@@ -2395,10 +2395,19 @@ public static function amend_header_image_with_video_header( $image_markup ) {
 				array_merge(
 					$video_attributes,
 					array(
-						'data-videoid'        => $youtube_id,
-						'data-param-rel'      => '0', // Don't show related videos.
-						'data-param-showinfo' => '0', // Don't show video title at the top.
-						'data-param-controls' => '0', // Don't show video controls.
+						'data-videoid'              => $youtube_id,
+
+						// For documentation on the params, see <https://developers.google.com/youtube/player_parameters>.
+						'data-param-rel'            => '0', // Don't show related videos.
+						'data-param-showinfo'       => '0', // Don't show video title at the top.
+						'data-param-controls'       => '0', // Don't show video controls.
+						'data-param-iv_load_policy' => '3', // Suppress annotations.
+						'data-param-modestbranding' => '1', // Show modest branding.
+						'data-param-playsinline'    => '1', // Prevent fullscreen playback on iOS.
+						'data-param-disablekb'      => '1', // Disable keyboard conttrols.
+						'data-param-fs'             => '0', // Suppress full screen button.
+						'data-param-loop'           => '1', // Loop video playback.
```

Comment:
Replace with `'data-param-loop'` with just `'loop'` and eliminate the subsequent `'data-param-playlist'` once https://github.com/ampproject/amphtml/pull/22858 is available.